packages/indexer-common/src/indexer-management/__tests__/resolvers/cost-models.ts

```diff
@@ -479,6 +489,72 @@ describe('Cost models', () => {
       [input],
     )
   })
+
+  test('Delete one cost model', async () => {
+    const inputs = [
+      {
+        deployment: '0x0000000000000000000000000000000000000000000000000000000000000000',
+        model: 'query { votes } => 10 * $n;',
+        variables: JSON.stringify({ n: 100 }),
+      },
+      {
+        deployment: '0x1111111111111111111111111111111111111111111111111111111111111111',
+        model: 'query { proposals } => 30 * $n;',
+        variables: JSON.stringify({ n: 10 }),
+      },
+    ]
+
+    for (const input of inputs) {
+      await client.mutation(SET_COST_MODEL_MUTATION, { costModel: input }).toPromise()
+    }
+
+    for (const input of inputs) {
+      await expect(
+        client.query(GET_COST_MODEL_QUERY, { deployment: input.deployment }).toPromise(),
+      ).resolves.toHaveProperty('data.costModel', input)
+    }
+
+    for (const input of inputs) {
+      await expect(
+        client
+          .mutation(DELETE_COST_MODELS_MUTATION, { deployments: [input.deployment] })
+          .toPromise(),
+      ).resolves.toHaveProperty('data.deleteCostModels', 1)
+    }
+  })
+
+  test('Delete all costs model', async () => {
+    const inputs = [
+      {
+        deployment: '0x0000000000000000000000000000000000000000000000000000000000000000',
+        model: 'query { votes } => 10 * $n;',
+        variables: JSON.stringify({ n: 100 }),
+      },
+      {
+        deployment: '0x1111111111111111111111111111111111111111111111111111111111111111',
+        model: 'query { proposals } => 30 * $n;',
+        variables: JSON.stringify({ n: 10 }),
+      },
+    ]
+
+    for (const input of inputs) {
+      await client.mutation(SET_COST_MODEL_MUTATION, { costModel: input }).toPromise()
+    }
+
+    for (const input of inputs) {
+      await expect(
+        client.query(GET_COST_MODEL_QUERY, { deployment: input.deployment }).toPromise(),
+      ).resolves.toHaveProperty('data.costModel', input)
+    }
+
+    await expect(
+      client
+        .mutation(DELETE_COST_MODELS_MUTATION, {
+          deployments: inputs.map((d) => d.deployment),
+        })
+        .toPromise(),
+    ).resolves.toHaveProperty('data.deleteCostModels', 2)
+  })
```

Comment:
Maybe add an additional test for deleting the `global` cost model?

Reply:
Added updated tests for `Set, get, and delete global cost model` and `Delete global costs model`